src/main/java/server/router/routes/MethodOptions.java

```diff
@@ -0,0 +1,29 @@
+package server.router.routes;
+
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+import server.constants.Method;
+import server.request.Request;
+import server.response.Response;
+import server.response.ResponseBuilder;
+
+public class MethodOptions implements RouteHandler {
+  @Override
+  public Set<Method> getMethods() {
+    return new LinkedHashSet<>(List.of(
+            Method.GET, Method.HEAD, Method.OPTIONS));
```

Comment:
Should we allow users to execute GET and HEAD requests to this endpoint? Similar question for `MethodOptions2`

Reply:
Users are able to execute all of the requests allowed for each of the endpoints and will receive a "HTTP/1.1 200 OK" response. If they send an Options request they'll also receive an Allow header. Should I be sending something different?

Reply:
I should have read the RFC first before sending my comment. Ignore me!

Reply:
Ok! Just making sure I wasn't overlooking something again!